docs/tctl/workflow/describe.md

```diff
@@ -9,17 +9,21 @@ tags:
 ---
 
 The `tctl workflow describe` command shows information about a [Workflow Execution](/concepts/what-is-a-workflow-execution).
+This information can be used to locate any deployment where a Workflow failed.
```

Comment:
What is a "deployment" here?
"... to locate any Workflow Execution that might have failed, for example"